lua/lspconfig/server_configurations/dockerls.lua

```diff
@@ -6,6 +6,15 @@ return {
     filetypes = { 'dockerfile' },
     root_dir = util.root_pattern 'Dockerfile',
     single_file_support = true,
+    settings = {
+      docker = {
+        languageserver = {
+          formatter = {
+            ignoreMultilineInstructions = true,
```

Comment:
it's better add an introduce of this config in docs/descripition

Reply:
moved, thanks for the fast feedback!